lib/alice/adapters.ex

```diff
@@ -0,0 +1,15 @@
+defmodule Alice.Adapters do
+  @moduledoc false
+
+  def start_link(opts \\ [start_adapter: true]) do
+    if opts[:start_adapter], do: selected_adapter.start_link
+    {:ok, selected_adapter}
+  end
+
+  @doc """
+  Selects adapter, defaults to Slack
+  """
+  def selected_adapter do
+    Application.get_env(:alice, :adapter, Alice.Adapters.Slack)
```

Comment:
@tapickell yes, that was a design decision that we made. However, there is nothing stopping you from spawning multiple instances of Alice, each running a different adapter. Being an OTP application, it can be part of a larger hierarchy of supervisors.